app/src/main/java/com/concordium/wallet/data/util/StorageAccessFrameworkUtil.kt

```diff
@@ -30,8 +30,9 @@ object StorageAccessFrameworkUtil {
 
                 // Note it's called "Display Name". This is
                 // provider-specific, and might not necessarily be the file name.
-                val displayName: String =
-                    it.getString(it.getColumnIndex(OpenableColumns.DISPLAY_NAME))
+
+                val index = it.getColumnIndex(OpenableColumns.DISPLAY_NAME)
+                val displayName: String = if (index == -1) "" else it.getString(index)
```

Comment:
Was this new behaviour? Why is using "" a sensible default? Is this not expected to happen? If so, why was the change necessary?

Reply:
In val index = it.getColumnIndex(OpenableColumns.DISPLAY_NAME) if the column doesn't exist it will return -1,
and it.getString(index) only accepts values for index >= 0 so this was flagged as error.
The value of displayName is just logged later in the code and is not used anywhere else:
Log.i("[SAF] Display Name: $displayName")
so if nothing found it will log something like this:
[SAF] Display Name:

Reply:
Ok.